### ISSUE TITLE
关于使用ray分布式运行ppo的问题

### ISSUE THREAD

**Issue by macheng6** [OpenRLHF/OpenRLHF#573]
我在2机16卡上训练，在head节点运行如下命令:
ray job submit --address="http://127.0.0.1:8265" \
   --runtime-env-json='{"working_dir": "/nas/xx/projects/OpenRLHF-main/openrlhf"}' \
   -- python openrlhf.cli.train_ppo_ray \
   --ref_num_nodes 1 \
   --ref_num_gpus_per_node 1 \
   --reward_num_nodes 1 \
   --reward_num_gpus_per_node 1 \
   --actor_num_nodes 1 \
   --actor_num_gpus_per_node 4 \
   --vllm_num_engines 2 \
   --vllm_tensor_parallel_size 1 \
   --pretrain /mnt4/ckpt/merge/v24_qwen2.5_7b \
   --reward_pretrain /mnt4/ckpt/merge/v5_qwen2.5_7b_rm \
   --save_path ${output_dir} \
   --micro_train_batch_size 8 \
   --train_batch_size 128 \
   --micro_rollout_batch_size 32 \
   --rollout_batch_size 1024 \
   --n_samples_per_prompt 1 \
   --max_epochs 1 \
   --prompt_max_len 1024 \
   --max_samples 100000 \
   --generate_max_len 1024 \
   --advantage_estimator reinforce \
   --zero_stage 3 \
   --bf16 \
   --actor_learning_rate 5e-7 \
   --critic_learning_rate 9e-6 \
   --init_kl_coef 0.01 \
   --prompt_data /nas/train/openrlhf_v24.json \
   --input_key messages \
   --apply_chat_template \
   --normalize_reward \
   --adam_offload \
   --gradient_checkpointing \
   --packing_samples \
   --save_steps -1

报错：
python: can't open file '/tmp/ray/session_2024-12-10_16-38-29_258673_264302/runtime_resources/working_dir_files/_ray_pkg_7b407da091e4f702/openrlhf.cli.train_ppo_ray': [Errno 2] No such file or directory

head节点有对应的文件，但是worker节点看了下，确实没有对应的文件，然后我copy到worker节点的对应目录下，运行依然报错，
想请教一下：
1. working_dir表示的是什么目录，如果openrlhf没有安装，是填写openrlhf的代码目录吗？如果openrlhf是安装在环境中，应该怎么填写。
2. 报错貌似是worker节点和head节点没有将一些信息同步，这个应该怎么解决。

**Comment by hijkzzz** [comment]
head 和 worker 应该共享网络磁盘
